### PR TITLE
Update meeting00.md

### PR DESCRIPTION
**Pull request by nspeer12**

### CODE REVIEW

File: fa20/09-10-meeting00/meeting00.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
-<!-- TODO Add Meeting Notes/Contents here -->
-<!-- NOTE Refer the Documentation if you're unsure how to format/add to this. -->
+# Project Brainstorm
```
Comment (jmuchovej) [inline fa20/09-10-meeting00/meeting00.md]: Forgot to state this – there should be no H1's in the `*.md` files. The website already has an H1 ready (it's the meeting title).

**Review by jmuchovej — CHANGES_REQUESTED** [review]
Forgot to state this – there should be no H1's in the `*.md` files. The website already has an H1 ready (it's the meeting title).

If you meant this to be the page-title, just drop it, `autobot` handles that.

**Review by nspeer12 — COMMENTED** [review]
fixed h1

**Review by jmuchovej — APPROVED** [review]

**Review by bb912 — APPROVED** [review]
See you there : )

**Pull request merged**